NEWS.txt

```diff
@@ -3,42 +3,40 @@ Release Notes
 =============
 
 
-Theano 0.10.0beta3 (20th of September, 2017)
-============================================
+Theano 0.10.0beta4 (16th of October, 2017)
+==========================================
 
 This release contains new features, improvements and bug fixes to prepare the upcoming release candidate.
 
 We recommend that every developer updates to this version.
 
 Highlights:
- - conda packages now available and updated in our own conda channel ``mila-udem``.
-   To install it: ``conda install -c mila-udem/label/pre theano pygpu``
-
- - Improved elemwise operations
-
-   - Speed-up elemwise ops based on SciPy
-   - Fixed memory leak related to elemwise ops on GPU
-
- - Improved pickling and tests in debug mode
- - Fixed pygpu detection
+ - Announcing that `MILA will stop developing Theano <https://groups.google.com/d/msg/theano-users/7Poq8BZutbY/rNCIfvAEAwAJ>`_
  - Bug fixes, crash fixes, warning improvements and documentation updates
 
-A total of 69 people contributed to this release since 0.9.0, see list below.
+A total of 70 people contributed to this release since 0.9.0, see list below.
 
 Interface changes:
- - Removed op ``ExtractDiag`` from ``theano.tensor.nlinalg``, now only in ``theano.tensor.basic``
+ - Generalized ``AllocDiag`` for any non-scalar input
 
 Convolution updates:
- - Added dilated causal convolutions for 2D
+ - Implemented fractional bilinear upsampling
+
+cuDNN (GPU):
+ - Disallowed ``float16`` precision for convolution gradients
+ - Fixed memory alignment detection
+ - Added profiling in C debug mode (with theano flag ``cmodule.debug=True``)
 
 New features:
- - Implemented ``topk`` and ``argtopk`` on CPU and GPU
- - Added ``unravel_index`` and ``ravel_multi_index`` functions on CPU
- - Implemented ``max()`` and ``min()`` functions for booleans and unsigned integers types
+ - Implemented truncated normal distribution with box-muller transform
+ - Added ``L_op()`` overriding option for ``OpFromGraph``
+ - Added NumPy C-API based fallback implementation for ``[sd]gemv_`` and ``[sd]dot_``
 
-Others:
- - Added ``R_op()`` for ``ZeroGrad``
- - Added description for rnnblock
+Other more detailed changes:
+ - Improved stack trace follow-up for GPU optimizations
+ - Fixed gradient error for elemwise ``minimum`` and ``maximum``
```

Comment:
specify when both values compared are the same.

Reply:
Fixed